### PR TITLE
Fix immutable.getIn and autofillPopupHidden condition

### LINKED ISSUE #6233: Autofill popup will not be dismissed
Version: https://github.com/brave/browser-laptop/commit/b9ec9a500c63b5475b7cefd9f5fc933835394037
https://github.com/brave/electron/commit/91d74aa4c2a35f1cb02264089218741251f457c7

STR:
1. Enter autofill data
2. Go to https://www.roboform.com/filling-test-all-fields
3. Double click the form and generate the popup
4. Press Esc/Tab won't make the popup disappear

### PR DESCRIPTION
**Pull request by darkdh**
- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).

Test Plan: Covered by auto test

fix #6233

Auditors: @bridiver, @bbondy

### CODE REVIEW

File: js/stores/windowStore.js [hunk 1]
```diff
@@ -658,8 +658,8 @@ const doAction = (action) => {
     case windowConstants.WINDOW_AUTOFILL_POPUP_HIDDEN:
     case windowConstants.WINDOW_SET_CONTEXT_MENU_DETAIL:
       if (!action.detail) {
-        if (windowState.getIn('contextMenuDetail', 'type') === 'autofill' &&
-            windowState.getIn('contextMenuDetail', 'tabId') === action.tabId) {
+        if (windowState.getIn(['contextMenuDetail', 'type']) === 'autofill' &&
+            windowState.getIn(['contextMenuDetail', 'tabId']) === action.tabId) {
```
Comment (bbondy) [inline js/stores/windowStore.js]: woops, it'd be nice to lint for this type of error eventually.

**Comment by bbondy** [comment]
++

File: js/components/frame.js [hunk 2]
```diff
@@ -688,7 +688,10 @@ class Frame extends ImmutableComponent {
       contextMenus.onShowAutofillMenu(e.suggestions, e.rect, this.frame)
     })
     this.webview.addEventListener('hide-autofill-popup', (e) => {
-      windowActions.autofillPopupHidden(this.props.tabId)
+      let webContents = this.webview.getWebContents()
```
Comment (bridiver) [inline js/components/frame.js]: this shouldn't be necessary anymore. I changed windowStore to make a check instead. I think we may want to revert this unless I'm missing something

File: js/stores/windowStore.js (hunk 1 above)
Comment (bridiver) [inline js/stores/windowStore.js]: this is probably the reason the code above was still needed. I think we can keep this and lose the `isFocused` check

**Pull request merged**